corehq/couchapps/__init__.py

```diff
@@ -7,8 +7,16 @@
     'users_extra': (settings.USERS_GROUPS_DB, settings.NEW_USERS_GROUPS_DB),
     'all_docs': (
         None, settings.NEW_USERS_GROUPS_DB, settings.NEW_FIXTURES_DB, 'meta',
-        settings.NEW_DOMAINS_DB),
+        settings.NEW_DOMAINS_DB, settings.NEW_APPS_DB),
     'by_domain_doc_type_date': (
         None, settings.NEW_USERS_GROUPS_DB, settings.NEW_FIXTURES_DB, 'meta',
-        settings.NEW_DOMAINS_DB),
+        settings.NEW_DOMAINS_DB, settings.NEW_APPS_DB),
+
+    # register these views to both the main db AND the apps db.
+    'app_translations_by_popularity': (None, settings.NEW_APPS_DB),
+    'exports_forms_by_app': (None, settings.NEW_APPS_DB),
+    'forms_by_app_info': (None, settings.NEW_APPS_DB),
+
+    # register just to the new apps db
+    'cloudcare_apps': settings.NEW_APPS_DB,
```

Comment:
However I think the plan I outlined here is simpler: https://github.com/dimagi/commcare-hq/pull/10022#issuecomment-174012914.

Rather than break `cloudcare/cloudcare_apps` out, just sync `_design/cloudcare` to apps db. (This is the only view in it, so it doesn't even need to be in the main db, just the apps db.) Once the migration's complete, we can then merge that other PR that merges this view with `app_manager/applications_brief`, and maximal optimality has been obtained